.github/dependabot.yml

```
@@ -142,11 +142,7 @@ updates:
       - dependency-name: org.eclipse.microprofile.opentracing:microprofile-opentracing-tck*
       - dependency-name: org.eclipse.microprofile.rest.client:microprofile-rest-client-tck
       # Dev UI web dependencies
-      - dependency-name: org.webjars:bootstrap
-      - dependency-name: org.webjars:font-awesome
-      - dependency-name: org.webjars:jquery
-      - dependency-name: org.webjars:codemirror
-      - dependency-name: org.webjars.npm:mermaid
+      - dependency-name: org.mvnpm:*
```

Shouldn't it be to also catch the Vaadin ones:
```suggestion
      - dependency-name: org.mvnpm:*
      - dependency-name: org.mvnpm.*:*
```